### DISCUSSION TITLE
Prácticas Front end

### DISCUSSION THREAD

**Discussion by T0ny-dev** [Launch-X-Latam/MisionFrontEnd#426] [Show and tell]
Hola buen día también anexo las prácticas de la Misión de Frontend

Repo: https://github.com/T0ny-dev/MisionFrontEnd-Launch-X-Latam
Deploy: https://t0ny-dev.github.io/MisionFrontEnd-Launch-X-Latam/

Mis mejores deseos estas fechas.

**Comment by kerizrh** [comment]
Wow, se nota que tienes mucha experiencia, me gustaron mucho tus prácticas. 

Por cierto, si aún queda un espacio en tu equipo para el hackathon me gustaría unirme.

**Reply by T0ny-dev** [reply]
si vente bro a darle, gracias :) seguimos mejorandolo y aprendiendo

**Reply by kerizrh** [reply]
Genial te dejo mi email para contactarnos: lx07570@innovaccion.mx 
o si gustas me pasas el tuyo y te escribo.
